src/components/Carousel/index.js

```diff
@@ -75,26 +75,63 @@ export default class Carousel extends PureComponent {
     highlightOnHover: false,
     loop: false,
     overflow: false,
-    scrollCount: 1,
+    scrollCount: 2,
     showCount: 3,
     transition: TRANSITION_SLIDE,
   }
 
+  state = {
+    currentSlide: 0,
+  }
+
+  constructor (props) {
+    super(props)
+
+    this.slider = props.sliderRef || React.createRef()
+  }
+
+  handlePrevClick = () => {
+    const { scrollCount } = this.props
+    const { currentSlide } = this.state
+
+    const goTo = currentSlide - scrollCount < 0
+      ? 0
+      : currentSlide - scrollCount
+
+    this.slider.current.slickGoTo(goTo)
+  }
+
+  handleNextClick = () => {
```

Comment:
Same here, would it be simpler / more readable to just `+1` to the `currentSlide` value?

Reply:
the number to increment is a prop and can be changed